src/main/scala/io/iohk/ethereum/network/PeerActor.scala

```diff
@@ -79,17 +79,17 @@ class PeerActor[R <: HandshakeResult](
         processHandshakerNextMessage(initHandshaker, rlpxConnection.copy(uriOpt = Some(uri)), numRetries)
 
       case RLPxConnectionHandler.ConnectionFailed =>
-        log.warning("Failed to establish RLPx connection")
+        log.debug("Failed to establish RLPx connection")
```

Comment:
as this class will only be logging in debug, isn't it better to set level OFF in  `universal/conf/logback.xml` which, if i'm not mistaken, is the config dist will use?